scoring_engine/models/service.py

```diff
@@ -11,7 +11,7 @@ class Service(Base):
     name = Column(String(100), nullable=False)
     check_name = Column(String(100), nullable=False)
     team_id = Column(Integer, ForeignKey('teams.id'))
-    team = relationship("Team", back_populates="services")
+    team = relationship("Team", back_populates="services", lazy="joined")
```

Comment:
Do we want to do this on all of these type of relationships across all the other models as well?

Reply:
I haven't found any other n+1 problems, and I don't necessarily want to eager load everything if we can avoid it.